WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -136,6 +136,8 @@ def getCPUTime( CPUNormalizationFactor ):
   if CPUTime:
     # This is in HS06sseconds
     # We need to convert in real seconds
+    if not CPUNormalizationFactor:
+      CPUNormalizationFactor = gConfig.getValue( '/LocalSite/CPUNormalizationFactor', 0.0 )
```

Comment:
This is exactly what it tries to avoid, in fact: the main client of this method is script dirac-wms-get-queue-cpu-time (used by the pilot) that by default passes in 0.0 as CPUNormalizationFactor. So in this case this utility will look to get it from the local cfg, but if it's not present even there, there's nothing to do, and yes at that point it will raise an exception. If this is run inside the pilot it will be safe.